app/namespace.go

```diff
@@ -184,6 +272,129 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 				return err
 			},
 			Subcommands: []*cli.Command{
+				{
+					Name:    "create",
+					Usage:   "Create a temporal namespace",
+					Aliases: []string{"c"},
+					Flags: []cli.Flag{
+						RequestIDFlag,
+						namespaceRegionFlag,
+						namespaceEnvironmentFlag,
+						&cli.StringFlag{
+							Name:     NamespaceFlagName,
+							Usage:    "The namespace hosted on temporal cloud",
+							Aliases:  []string{"n"},
+							Required: true,
+						},
+						&cli.IntFlag{
+							Name:    RetentionDaysFlagName,
+							Usage:   "The retention of the namespace in days",
+							Aliases: []string{"rd"},
+							Value:   30,
+						},
+						&cli.StringFlag{
+							Name:    CaCertificateFlagName,
```

Comment:
There is an existing flag that we could reuse: https://github.com/temporalio/tcld/blob/main/app/namespace.go#L26

Reply:
will reuse this one.